system/jlib/jsocket.cpp

```diff
@@ -3946,9 +3945,9 @@ class CSocketBaseThread: public Thread
             return true;
         else if (rc>0)
         {
-            if ( !(fds[0].revents & POLLNVAL) )
+            if ( !(fds[0].revents & POLLNVAL) ) // TODO: MCK - also check POLLERR here ?
             {
-                PROGLOG("CSocketBaseThread: poll handle %d selected(2) %d",sock,rc);
+                // PROGLOG("CSocketBaseThread: poll handle %d selected(2) %d",sock,rc);
```

Comment:
should delete commented lines or *perhaps* put in a ifdef block, but if deemed not needed at all, better to delete.

Reply:
Put it in an ifdef _DEBUG